### ISSUE TITLE
404 Problem https://api.arte.tv/api/pl ayer/v1/config/fr/....

### ISSUE THREAD

**Issue by eric-blabla** [Catch-up-TV-and-More/plugin.video.catchuptvandmore#851]
Hi, 

I have a problem with arte tv: here is the log entry: 

ERROR: [Catch-up-TV-&-More.support] 404 Client Error: Not Found for url: 
https://api.arte.tv/api/player/v1/config/fr/102958-014-A

I think that the plugin is up-to-date, so I do not know what to do. Thank you for any help!

**Comment by joaopa00** [comment]
This problem is fixed in the latest beta. You have to install it.

**Comment by eric-blabla** [comment]
Now I get "unable to import route module" : resources.lib.main

as a result,  CU TV & more  is dead ... hell!

**Comment by joaopa00** [comment]
Are you using Kodi 18 Leia?

**Comment by eric-blabla** [comment]
yes, I do. Please don't tell me I need kodi 19 - I have a rasp3 and there is no such thing as kodi19 for it ...

**Comment by joaopa00** [comment]
No. This bug is fixed in the latest git. If you want to give a  try, you can download the git. Otherwise, you have just to wait the release of the beta, that could happen the next week.

**Comment by eric-blabla** [comment]
Ouff!  I am willing to test, of course! 
So I git clone CUTV on my computer, and then push it ivia scp into kodi. But in which path? Sorry for asking these beginner questions.  Thank you!

**Comment by ThTetras** [comment]
Même problème que eric-blabla... 
Ma config sur Raspi3 
Debian GNU/Linux 10 (buster)
OSMC 2022.03-1
Kodi (19.4)
Je suis dans le canal Beta (dans le dépôt je n'ai que le 0.0.6), et je ne peux charger que la version 0.2.38-beta01
arte ne fonctionne pas en rattrapage mais fonctionne bien en direct.
Il y a une manip à faire pour obtenir la beta la plus récente ?
Merci pour votre travail, CUTV&+ est vraiment bien !

**Comment by ThTetras** [comment]
Je dois peut-être l'installer depuis un zip téléchargé comme
https://github.com/Catch-up-TV-and-More/repository.catchuptvandmore/blob/master/zips/beta/plugin.video.catchuptvandmore/plugin.video.catchuptvandmore-0.2.38%7Ebeta28.zip
Je viens de le tenter... Je redémarre... 
Et ça le fait !!! Sympa !!!
Merci encore

**Comment by joaopa00** [comment]
J'ai libéré la dernière béta qui devrait fixer tous vos problèmes. Pour installer le dernière béta, il faut lire le wiki qui est ici:
[en français](https://catch-up-tv-and-more.github.io/fr/installation/)

I released the latest beta. Normally, it should fix all your problem. To install it, please have a look at
[in english](https://catch-up-tv-and-more.github.io/installation/)

**Comment by eric-blabla** [comment]
I have  version 02.38 beta 28. Now I get 

"unable to import route moduleresources.lib.main"

I remember that I had that, maybe a coding system problem? Or Inputstream? (adaptive=2.4.8.1 helper=0.5.10)

**Comment by eric-blabla** [comment]
tried complete removal & reinstall. Did not help :((

**Comment by joaopa00** [comment]
The latest version is beta-29.

**Comment by eric-blabla** [comment]
WGET don't work. I get an html instead... my fault. Now:

2022-05-16 20:03:31.691 T:1807721344  NOTICE: ADDON: audioencoder.kodi.builtin.aac v1.0.0 installed
2022-05-16 20:03:31.692 T:1807721344  NOTICE: ADDON: audioencoder.kodi.builtin.wma v1.0.0 installed
2022-05-16 20:03:31.692 T:1807721344  NOTICE: ADDON: catchuptvandmore.kodi.beta v0.0.6 installed
2022-05-16 20:03:31.692 T:1807721344  NOTICE: ADDON: driver.dvb.crazycat v9.2.0.0 installed
2022-05-16 20:03:31.692 T:1807721344  NOTICE: ADDON: driver.dvb.dvb-latest v9.2.0.0 installed
2022-05-16 20:03:31.692 T:1807721344  NOTICE: ADDON: game.controller.default v1.0.19 installed
2022-05-16 20:03:31.692 T:1807721344  NOTICE: ADDON: game.controller.snes v1.0.18 installed
2022-05-16 20:03:31.692 T:1807721344  NOTICE: ADDON: inputstream.adaptive v2.4.8.1 installed
2022-05-16 20:03:31.692 T:1807721344  NOTICE: ADDON: kodi.binary.global.audioengine v1.0.1 installed
2022-05-16 20:03:31.692 T:1807721344  NOTICE: ADDON: kodi.binary.global.filesystem v1.0.2 installed
2022-05-16 20:03:31.692 T:1807721344  NOTICE: ADDON: kodi.binary.global.general v1.0.3 installed
2022-05-16 20:03:31.692 T:1807721344  NOTICE: ADDON: kodi.binary.global.gui v5.12.0 installed
2022-05-16 20:03:31.692 T:1807721344  NOTICE: ADDON: kodi.binary.global.main v1.0.14 installed
2022-05-16 20:03:31.692 T:1807721344  NOTICE: ADDON: kodi.binary.global.network v1.0.0 installed
2022-05-16 20:03:31.692 T:1807721344  NOTICE: ADDON: kodi.binary.instance.audiodecoder v2.0.0 installed
2022-05-16 20:03:31.692 T:1807721344  NOTICE: ADDON: kodi.binary.instance.audioencoder v2.0.0 installed
2022-05-16 20:03:31.692 T:1807721344  NOTICE: ADDON: kodi.binary.instance.game v1.1.0 installed
2022-05-16 20:03:31.692 T:1807721344  NOTICE: ADDON: kodi.binary.instance.imagedecoder v2.0.0 installed
2022-05-16 20:03:31.692 T:1807721344  NOTICE: ADDON: kodi.binary.instance.inputstream v2.0.8 installed
2022-05-16 20:03:31.692 T:1807721344  NOTICE: ADDON: kodi.binary.instance.peripheral v1.3.7 installed
2022-05-16 20:03:31.692 T:1807721344  NOTICE: ADDON: kodi.binary.instance.pvr v5.10.3 installed
2022-05-16 20:03:31.692 T:1807721344  NOTICE: ADDON: kodi.binary.instance.screensaver v2.0.0 installed
2022-05-16 20:03:31.692 T:1807721344  NOTICE: ADDON: kodi.binary.instance.vfs v2.0.0 installed
2022-05-16 20:03:31.692 T:1807721344  NOTICE: ADDON: kodi.binary.instance.videocodec v1.0.1 installed
2022-05-16 20:03:31.692 T:1807721344  NOTICE: ADDON: kodi.binary.instance.visualization v2.0.1 installed
2022-05-16 20:03:31.692 T:1807721344  NOTICE: ADDON: kodi.resource v1.0.0 installed
2022-05-16 20:03:31.693 T:1807721344  NOTICE: ADDON: metadata.album.universal v3.1.7 installed
2022-05-16 20:03:31.693 T:1807721344  NOTICE: ADDON: metadata.artists.universal v4.3.7 installed
2022-05-16 20:03:31.693 T:1807721344  NOTICE: ADDON: metadata.common.allmusic.com v3.2.2 installed
2022-05-16 20:03:31.693 T:1807721344  NOTICE: ADDON: metadata.common.fanart.tv v3.6.4 installed
2022-05-16 20:03:31.693 T:1807721344  NOTICE: ADDON: metadata.common.imdb.com v3.2.6 installed
2022-05-16 20:03:31.693 T:1807721344  NOTICE: ADDON: metadata.common.musicbrainz.org v2.2.4 installed
2022-05-16 20:03:31.693 T:1807721344  NOTICE: ADDON: metadata.common.theaudiodb.com v2.0.3 installed
2022-05-16 20:03:31.693 T:1807721344  NOTICE: ADDON: metadata.common.themoviedb.org v3.2.18 installed
2022-05-16 20:03:31.693 T:1807721344  NOTICE: ADDON: metadata.local v1.0.0 installed
2022-05-16 20:03:31.693 T:1807721344  NOTICE: ADDON: metadata.themoviedb.org v5.2.6 installed
2022-05-16 20:03:31.693 T:1807721344  NOTICE: ADDON: metadata.tvshows.themoviedb.org v3.5.14 installed
2022-05-16 20:03:31.693 T:1807721344  NOTICE: ADDON: os.libreelec.tv v9.2 installed
2022-05-16 20:03:31.693 T:1807721344  NOTICE: ADDON: os.openelec.tv v9.2 installed
2022-05-16 20:03:31.693 T:1807721344  NOTICE: ADDON: peripheral.joystick v1.4.9 installed
2022-05-16 20:03:31.693 T:1807721344  NOTICE: ADDON: plugin.program.chrome.launcher v1.2.0 installed
2022-05-16 20:03:31.693 T:1807721344  NOTICE: ADDON: plugin.video.ardundzdf v3.6.3 installed
2022-05-16 20:03:31.693 T:1807721344  NOTICE: ADDON: plugin.video.arteplussept v1.0.2 installed
2022-05-16 20:03:31.693 T:1807721344  NOTICE: ADDON: plugin.video.catchuptvandmore v0.2.38~beta29 installed
2022-05-16 20:03:31.693 T:1807721344  NOTICE: ADDON: plugin.video.francetv v2.0.1 installed
2022-05-16 20:03:31.693 T:1807721344  NOTICE: ADDON: plugin.video.mubi v1.1.1 installed
2022-05-16 20:03:31.693 T:1807721344  NOTICE: ADDON: plugin.video.netflix v1.12.9 installed
2022-05-16 20:03:31.693 T:1807721344  NOTICE: ADDON: plugin.video.sandmann v2.1.0 installed
2022-05-16 20:03:31.693 T:1807721344  NOTICE: ADDON: plugin.video.tester v0.2.0 installed
2022-05-16 20:03:31.693 T:1807721344  NOTICE: ADDON: plugin.video.youtube v6.8.18 installed
2022-05-16 20:03:31.693 T:1807721344  NOTICE: ADDON: pvr.iptvsimple v3.10.1.1 installed
2022-05-16 20:03:31.693 T:1807721344  NOTICE: ADDON: repository.castagnait v1.0.1 installed
2022-05-16 20:03:31.693 T:1807721344  NOTICE: ADDON: repository.flxapps v1.0.0 installed
2022-05-16 20:03:31.693 T:1807721344  NOTICE: ADDON: repository.kodi.game v1.0.0 installed
2022-05-16 20:03:31.694 T:1807721344  NOTICE: ADDON: repository.libreelec.tv v9.2.0 installed
2022-05-16 20:03:31.694 T:1807721344  NOTICE: ADDON: repository.sandmann79.plugins v1.0.4 installed
2022-05-16 20:03:31.694 T:1807721344  NOTICE: ADDON: repository.sendtokodi v0.0.1 installed
2022-05-16 20:03:31.694 T:1807721344  NOTICE: ADDON: repository.slyguy v0.0.3 installed
2022-05-16 20:03:31.694 T:1807721344  NOTICE: ADDON: repository.xbmc.org v3.1.6 installed
2022-05-16 20:03:31.694 T:1807721344  NOTICE: ADDON: resource.images.catchuptvandmore v1.0.14 installed
2022-05-16 20:03:31.694 T:1807721344  NOTICE: ADDON: resource.images.weathericons.default v1.1.8 installed
2022-05-16 20:03:31.694 T:1807721344  NOTICE: ADDON: resource.language.en_gb v2.0.1 installed
2022-05-16 20:03:31.694 T:1807721344  NOTICE: ADDON: resource.uisounds.kodi v1.0.0 installed
2022-05-16 20:03:31.694 T:1807721344  NOTICE: ADDON: screensaver.xbmc.builtin.black v1.0.33 installed
2022-05-16 20:03:31.694 T:1807721344  NOTICE: ADDON: screensaver.xbmc.builtin.dim v1.0.59 installed
2022-05-16 20:03:31.694 T:1807721344  NOTICE: ADDON: script.audio.profiles v2.0.0 installed
2022-05-16 20:03:31.694 T:1807721344  NOTICE: ADDON: script.module.addon.signals v0.0.6 installed
2022-05-16 20:03:31.694 T:1807721344  NOTICE: ADDON: script.module.backports.functools_lru_cache v1.6.1 installed
2022-05-16 20:03:31.694 T:1807721344  NOTICE: ADDON: script.module.beautifulsoup4 v4.9.3 installed
2022-05-16 20:03:31.694 T:1807721344  NOTICE: ADDON: script.module.certifi v2019.9.11 installed
2022-05-16 20:03:31.694 T:1807721344  NOTICE: ADDON: script.module.chardet v3.0.4 installed
2022-05-16 20:03:31.694 T:1807721344  NOTICE: ADDON: script.module.codequick v1.0.2 installed
2022-05-16 20:03:31.694 T:1807721344  NOTICE: ADDON: script.module.dateutil v2.8.1 installed
2022-05-16 20:03:31.694 T:1807721344  NOTICE: ADDON: script.module.future v0.17.1 installed
2022-05-16 20:03:31.694 T:1807721344  NOTICE: ADDON: script.module.htmlement v1.0.0 installed
2022-05-16 20:03:31.694 T:1807721344  NOTICE: ADDON: script.module.idna v2.8 installed
2022-05-16 20:03:31.694 T:1807721344  NOTICE: ADDON: script.module.inputstreamhelper v0.5.10 installed
2022-05-16 20:03:31.694 T:1807721344  NOTICE: ADDON: script.module.kodi-six v0.1.3 installed
2022-05-16 20:03:31.694 T:1807721344  NOTICE: ADDON: script.module.lib3sat v1.0.0 installed
2022-05-16 20:03:31.694 T:1807721344  NOTICE: ADDON: script.module.libard v1.0.0 installed
2022-05-16 20:03:31.694 T:1807721344  NOTICE: ADDON: script.module.libbr v1.0.0 installed
2022-05-16 20:03:31.694 T:1807721344  NOTICE: ADDON: script.module.libkika v1.0.0 installed
2022-05-16 20:03:31.694 T:1807721344  NOTICE: ADDON: script.module.libmdr v1.0.0 installed
2022-05-16 20:03:31.695 T:1807721344  NOTICE: ADDON: script.module.libmediathek3 v1.5.0 installed
2022-05-16 20:03:31.695 T:1807721344  NOTICE: ADDON: script.module.libzdf v2.1.7 installed
2022-05-16 20:03:31.695 T:1807721344  NOTICE: ADDON: script.module.mechanicalsoup v1.0.0 installed
2022-05-16 20:03:31.695 T:1807721344  NOTICE: ADDON: script.module.mechanize v0.2.6 installed
2022-05-16 20:03:31.695 T:1807721344  NOTICE: ADDON: script.module.mediathekxmlservice v1.0.0 installed
2022-05-16 20:03:31.695 T:1807721344  NOTICE: ADDON: script.module.pil v1.1.7 installed
2022-05-16 20:03:31.695 T:1807721344  NOTICE: ADDON: script.module.pyautogui v0.9.51 installed
2022-05-16 20:03:31.695 T:1807721344  NOTICE: ADDON: script.module.pycryptodome v3.4.3 installed
2022-05-16 20:03:31.695 T:1807721344  NOTICE: ADDON: script.module.pyqrcode v0.0.2 installed
2022-05-16 20:03:31.695 T:1807721344  NOTICE: ADDON: script.module.pytz v2014.2 installed
2022-05-16 20:03:31.695 T:1807721344  NOTICE: ADDON: script.module.pyxbmct v1.3.1 installed
2022-05-16 20:03:31.695 T:1807721344  NOTICE: ADDON: script.module.requests v2.22.0 installed
2022-05-16 20:03:31.695 T:1807721344  NOTICE: ADDON: script.module.six v1.15.0 installed
2022-05-16 20:03:31.695 T:1807721344  NOTICE: ADDON: script.module.slyguy v0.52.5 installed
2022-05-16 20:03:31.695 T:1807721344  NOTICE: ADDON: script.module.soupsieve v1.9.6+leia.1 installed
2022-05-16 20:03:31.695 T:1807721344  NOTICE: ADDON: script.module.tzlocal v2.0.0 installed
2022-05-16 20:03:31.695 T:1807721344  NOTICE: ADDON: script.module.urllib3 v1.25.6 installed
2022-05-16 20:03:31.695 T:1807721344  NOTICE: ADDON: script.module.xbmcswift2 v13.0.4 installed
2022-05-16 20:03:31.695 T:1807721344  NOTICE: ADDON: script.module.youtube.dl v21.303.0 installed
2022-05-16 20:03:31.695 T:1807721344  NOTICE: ADDON: script.program.driverselect v0.1.6 installed
2022-05-16 20:03:31.695 T:1807721344  NOTICE: ADDON: script.web.viewer v0.9.21 installed
2022-05-16 20:03:31.695 T:1807721344  NOTICE: ADDON: service.iptv.manager v0.2.3 installed
2022-05-16 20:03:31.695 T:1807721344  NOTICE: ADDON: service.libreelec.settings v9.0.0 installed
2022-05-16 20:03:31.695 T:1807721344  NOTICE: ADDON: service.locale v9.2.0.102 installed
2022-05-16 20:03:31.695 T:1807721344  NOTICE: ADDON: service.subtitles.opensubtitles v5.0.16 installed
2022-05-16 20:03:31.695 T:1807721344  NOTICE: ADDON: skin.estuary v2.0.27 installed
2022-05-16 20:03:31.695 T:1807721344  NOTICE: ADDON: slyguy.dependencies v0.0.5 installed
2022-05-16 20:03:31.695 T:1807721344  NOTICE: ADDON: slyguy.disney.plus v0.11.7 installed
2022-05-16 20:03:31.696 T:1807721344  NOTICE: ADDON: webinterface.default v18.x-2.4.6 installed
2022-05-16 20:03:31.696 T:1807721344  NOTICE: ADDON: xbmc.addon v18.9 installed
2022-05-16 20:03:31.696 T:1807721344  NOTICE: ADDON: xbmc.core v0.1.0 installed
2022-05-16 20:03:31.696 T:1807721344  NOTICE: ADDON: xbmc.gui v5.14.0 installed
2022-05-16 20:03:31.696 T:1807721344  NOTICE: ADDON: xbmc.json v10.3.0 installed
2022-05-16 20:03:31.696 T:1807721344  NOTICE: ADDON: xbmc.metadata v2.1.0 installed
2022-05-16 20:03:31.696 T:1807721344  NOTICE: ADDON: xbmc.python v2.26.0 installed
2022-05-16 20:03:31.696 T:1807721344  NOTICE: ADDON: xbmc.webinterface v1.0.0 installed
2022-05-16 20:03:33.745 T:1799328640   ERROR: EXCEPTION Thrown (PythonToCppException) : -->Python callback/script returned the following error<--

**Comment by joaopa00** [comment]
The repo is synchronized. You should try to install from the repo.

**Comment by eric-blabla** [comment]
Did it. I removed *all* CUTV traces (repo, addons), clear table.
Then reinstall repo, install (beta29) freom repo. Same error.

2022-05-16 22:41:55.222 T:1585939328   ERROR: EXCEPTION Thrown (PythonToCppException) : -->Python callback/script returned the following error<--

despite this error, the addon is installed. I an run it.
-> LiveTV: "route missing" error : details here  https://paste.kodi.tv/uyebuzawiv
-> CatchUp:  same. "route msiisng: resources.lib.channel.... " error

Getting desparate

**Comment by joaopa00** [comment]
Did you install the locale addon?

**Comment by eric-blabla** [comment]
yes, 9.2.0.102 is installed. I re-installed it, one never knows.

**Comment by darodi** [comment]
```
2022-05-16 22:27:46.559 T:1577546624   ERROR: GetDirectory - Error getting catch-up-tv-and-more.github.io/repo/
2022-05-16 22:32:37.040 T:1937158208   ERROR: Previous line repeats 1 times.
2022-05-16 22:32:37.040 T:1937158208   ERROR: CCurlFile::Open failed with code 404 for https://catch-up-and-more.github.io/repo:
2022-05-16 22:32:37.040 T:1937158208   ERROR: GetDirectory - Unable to get http directory (https://catch-up-and-more.github.io/repo)
2022-05-16 22:32:37.040 T:1937158208   ERROR: GetDirectory - Error getting https://catch-up-and-more.github.io/repo
```

first, correct your repository url:

`https://catch-up-and-more.github.io/repo`
--->
`https://catch-up-tv-and-more.github.io/repo/`

then reinstall with the repository and tell us if you still have problems.

**Comment by eric-blabla** [comment]
thank you darodi. I had that typo, but of course you cannot install the repo with it. At that stage I did not know how to modify the source file, so I made a new one, and installed from that one (this is what happened at May 16).

Anyways,  I still followed you instructions: 
1) learned the "c" key meaning and removed all wrong / old source files   :) THEN
2) deleted all CUTV (repo, addon, also the image whatever part)   and re-installed beta29 from new.

Problems persist, see https://paste.kodi.tv/rinugafuxe

**Comment by eric-blabla** [comment]
You frighten me, joaopa00 ! I mean:  

De-installing kodi from libreelec not sure I can do that. That means rather "clean new flash"  ...
What frightens me is that it took literally hours to make netflix and disney+  work. If they fail, I get a medium-size marriage crisis :)) Half joking.  

If I could  backup those two ( & restore them)  I had little scruple to reflash libreelec from scratch. 
But how can I ensure that?    (I admit that is a question out-of-CUTV scope), but maybe you know the answer by chance?

best, Eric

**Comment by darodi** [comment]
You are missing all the dependencies. 
I don't know how you got into this situation
They should be installed automatically if you use the repository beta

- `2022-05-18 22:09:37.072 T:1418707840   ERROR: GetDirectory - Error getting /usr/lib/kodi/addons/inputstream.adaptive/`
you should reinstall inputstream.adaptive

- `2022-05-18 22:08:29.080 T:1807721344  NOTICE: LoadJpeg: unable to load resource://resource.images.catchuptvandmore/channels/fr/bfmtv_fanart.jpg`
 you should reinstall the project https://github.com/Catch-up-TV-and-More/resource.images.catchuptvandmore

Don't mess with the source files, just use the repository beta and install all dependencies.

**Comment by joaopa00** [comment]
Assuming fixed. If the bug still occurs, please reopen.
